plugins/cost-insights/src/alerts/ProjectGrowthAlert.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { ProjectGrowthAlertCard } from '../components/ProjectGrowthAlertCard';
+import { Alert, ProjectGrowthData } from '../types';
+
+/**
+ * The alert below is an example of an Alert implementation; the CostInsightsApi permits returning
+ * any implementation of the Alert type, so adopters can create their own. The CostInsightsApi
+ * fetches alert data from the backend, then creates Alert classes with the data.
+ */
+
+export class ProjectGrowthAlert implements Alert {
+  data: ProjectGrowthData;
+
+  url = '/cost-insights/investigating-growth';
+  subtitle =
+    'Cost growth outpacing business growth is unsustainable long-term.';
+
+  constructor(data: ProjectGrowthData) {
+    this.data = data;
+  }
+
+  get title() {
+    return `Investigate cost growth in project ${this.data.project}`;
+  }
+
+  get element() {
+    return <ProjectGrowthAlertCard alert={this.data} />;
+  }
+}
```

Comment:
Out of scope for this feature, but I wonder how much utility these out-of-the-box alerts offer in their current state.

If an implementor wanted to support alert dismissal for project growth alerts, for example, they could extend the prototype of `ProjectGrowthAlert` (since no onDismissed hook is defined) but if they also wanted to override the default url, would have to resort to some hacky workarounds - like manually overriding the url property or using getters to intercept access.

One possible idea is to convert these to [abstract classes](https://www.typescriptlang.org/docs/handbook/classes.html#abstract-classes).